buildSrc/private/src/main/kotlin/androidx/build/AndroidXImplPlugin.kt

```diff
@@ -172,15 +172,12 @@ class AndroidXImplPlugin @Inject constructor(val componentFactory: SoftwareCompo
 //            }
 //        }
 
-        // We Use languageVersion 1.9 for now to align with Jetpack Compose
```

Comment:
Why is it not the case anymore? Probably it makes sense for JVM/Desktop

Reply:
cc: @igordmn Do we want LV 1_9 for desktop?

Reply:
If we don't have issues with LV 1_9, better to keep it.

Reply:
Fixed